install/kots/manifests/gitpod-installer-job.yaml

```diff
@@ -43,7 +43,7 @@ spec:
       containers:
         - name: installer
           # This will normally be the release tag - using this tag as need the license evaluator
-          image: 'eu.gcr.io/gitpod-core-dev/build/installer:main.2569'
+          image: 'eu.gcr.io/gitpod-core-dev/build/installer:sje-kots-helm.4'
```

Comment:
So, There will be a separate PR to update this back to main? :thinking:

Reply:
As part of the next release process, this will get updated to whatever the Docker image is - see [2022.02.0](https://github.com/gitpod-io/gitpod/releases/tag/2022.02.0) for example